lms/envs/common.py

```diff
@@ -1261,17 +1261,15 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'edx-pattern-library/js/modernizr-custom.js',
```

Comment:
I've included Modernizr as a globally loaded script, because it's necessary for the AFontGarde icon font solution. Since icon fonts could (and probably are) used on every page, I felt it useful to make Modernizr globally available as well.